src/libraries/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -42,11 +42,15 @@ public abstract partial class Delegate : ICloneable, ISerializable
         public static Delegate CreateDelegate(Type type, MethodInfo method) => CreateDelegate(type, method, throwOnBindFailure: true)!;
 
         // V1 api: Creates closed delegates to instance methods only, relaxed signature checking disallowed.
+        [RequiresUnreferencedCode("The target method might be removed")]
```

Comment:
What about ref APIs? Shouldn't they be updated as well?

Reply:
The tooling is crapping out and I haven't yet reverse engineered how to make it do what it's supposed to. See also https://github.com/dotnet/runtime/pull/37741#issuecomment-642541879 and #37966 that I just filed.

I'll do it in a single batch at a later time since manually updating APIs with so many overloads is bound to introduce bugs.